src/sentry/api/event_search.py

```diff
@@ -919,6 +920,26 @@ def resolve_orderby(orderby, fields, aggregations):
     validated = []
     for column in orderby:
         bare_column = column.lstrip("-")
+        if bare_column == SORTED_PROJECT_SLUG:
```

Comment:
Do we need the suffixes version in the url/parameters? Isn't project and project.name enough signal to apply this logic?